### PR TITLE
View separate page for each topic

### PR DESCRIPTION
**Pull request by gwdawson**

### CODE REVIEW

File: src/components/Nav.jsx [hunk 1]
```diff
@@ -8,13 +8,13 @@ export default function Nav() {
...
-      <Link className='NavChild' to='/'>
+      <Link className='NavChild' to='/topics/coding'>
         Coding
       </Link>
-      <Link className='NavChild' to='/'>
+      <Link className='NavChild' to='/topics/football'>
         Football
       </Link>
-      <Link className='NavChild' to='/'>
+      <Link className='NavChild' to='/topics/cooking'>
```
Comment (karlaferry) [inline src/components/Nav.jsx:17]: Honestly would suggest not hardcoding the topics like this and make it dynamic. If I remember correctly your backend should be able to serve just the list of topics, so you could make an API call to request all that.

**Review by karlaferry — CHANGES_REQUESTED** [review]
Make it more dynamic and account for the possibility of a user being able to add a new topic. :)

File: src/components/Header.jsx [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
-  return <h1 className='Header'>All Articles</h1>;
+  const { pathname } = useLocation();
+  let topic = undefined;
+
+  if (pathname === '/') topic = 'All';
+  else topic = pathname.split('/')[2];
+
+  const first = topic[0].toUpperCase();
+  const rest = topic.slice(1);
+  const newTopic = first + rest;
```
Comment (jadelandeg) [inline src/components/Header.jsx:14]: what are these variables for?

**Review by jadelandeg — CHANGES_REQUESTED** [review]
Looks really good! It looks like you have some unused variables. The first and rest ones. What are they doing?

File: src/components/Header.jsx (hunk 2 above)
Comment (gwdawson) [reply src/components/Header.jsx:14]: I'm sending `newTopic` on line 16.
this is just capitalising the first letter of the header.
`coding Articles` -> `Coding Articles`

**Review by jadelandeg — APPROVED** [review]

**Pull request merged**